### ISSUE TITLE
QR code event repeating after 1 minute

### ISSUE THREAD

**Issue by stefanfuchs** [pedroslopez/whatsapp-web.js#645]
### Bug description
The QR codes (received via the 'qr' event) are being repeated after 1 minute from when the whatsapp-web.js client is launched.
This is a new bug that started today, as far as I know.
In the logs bellow, the repeated QR codes are shown. They continue like that for a long time.

```
[2021-04-20 15:26:57] 2021-04-20T18:26:57.151Z ChatBotWhatsappServer QR RECEIVED 1@+5DztSkYQicV6vvXxtQunNWK95t0ONKQn8oD7wripRfDUBOVYJsaAeuXERzyJxJJy+x0jpYspCQ3ZQ==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
[2021-04-20 15:27:17] 2021-04-20T18:27:17.069Z ChatBotWhatsappServer QR RECEIVED 1@ldVLzuCxfEGZBub7p+Kq9QEVbh+7Mo9I0qbcPO6bno8njSfoMMjwVhRRo+kBGocDH9lamRtKyVdWww==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
[2021-04-20 15:27:37] 2021-04-20T18:27:37.037Z ChatBotWhatsappServer QR RECEIVED 1@Ku6bZbaqah9ap4RlbDYcAEx2DvtijstOdjEAFR2oOBp0bOvrksifujUKih+gLrDp86Vby2ScZZu2EA==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
[2021-04-20 15:27:57] 2021-04-20T18:27:57.021Z ChatBotWhatsappServer QR RECEIVED 1@5G3CyE23VAONeS+WAKKF1+V01oETyMrtbRkDrRghBDshopLL7hIjVKroqH68dXchEnSVRyy7FEuS6w==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
[2021-04-20 15:28:17] 2021-04-20T18:28:17.134Z ChatBotWhatsappServer QR RECEIVED 1@svuDd8chbTr77d68Y1L3mOhGMU3F/C9aHnh8RUbg1QlsVbW2tyYHsavfJ48YHdZoLqItjw2eaRL4rg==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
[2021-04-20 15:28:37] 2021-04-20T18:28:37.029Z ChatBotWhatsappServer QR RECEIVED 1@7LdUUqR55Oy2UuDNjVkrFzNimSMxW0YPDJAvGYKGEdQK1a7jWmlp7s+aT9715GcFrq8tGX65n2sPZw==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
[2021-04-20 15:28:57] 2021-04-20T18:28:57.056Z ChatBotWhatsappServer QR RECEIVED 1@7LdUUqR55Oy2UuDNjVkrFzNimSMxW0YPDJAvGYKGEdQK1a7jWmlp7s+aT9715GcFrq8tGX65n2sPZw==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
[2021-04-20 15:29:17] 2021-04-20T18:29:17.028Z ChatBotWhatsappServer QR RECEIVED 1@7LdUUqR55Oy2UuDNjVkrFzNimSMxW0YPDJAvGYKGEdQK1a7jWmlp7s+aT9715GcFrq8tGX65n2sPZw==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
[2021-04-20 15:29:37] 2021-04-20T18:29:37.052Z ChatBotWhatsappServer QR RECEIVED 1@7LdUUqR55Oy2UuDNjVkrFzNimSMxW0YPDJAvGYKGEdQK1a7jWmlp7s+aT9715GcFrq8tGX65n2sPZw==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
[2021-04-20 15:29:57] 2021-04-20T18:29:57.032Z ChatBotWhatsappServer QR RECEIVED 1@7LdUUqR55Oy2UuDNjVkrFzNimSMxW0YPDJAvGYKGEdQK1a7jWmlp7s+aT9715GcFrq8tGX65n2sPZw==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
[2021-04-20 15:30:17] 2021-04-20T18:30:17.041Z ChatBotWhatsappServer QR RECEIVED 1@7LdUUqR55Oy2UuDNjVkrFzNimSMxW0YPDJAvGYKGEdQK1a7jWmlp7s+aT9715GcFrq8tGX65n2sPZw==,kbK0TlHDu0WWPXXbh4bUatee/IidKLZpGF/GY5riEzE=,iHxF+o/6e+7ixXajE6FF5Q==
```

### Reproduction steps
1. Launch whatsapp-web.js without previous authentication
2. Observe the received QR codes (via the 'qr' event), and wait 1 minute

### Expected behavior 
New QR codes should be received every time.

### Relevant code
```js
whatsappClient.on('qr', (qr) => console.log(qr))
```

### Environment (please complete the following information):
**WhatsApp**
 - Account type [Standard / Business]: Business
 - Device OS [iOS / Android]: Android
 - WhatsApp Web version [run `await client.getWWebVersion()`]: v2.2114.8

**Library**
 - Browser [Chrome / Chromium]: Chromium
 - NodeJS version (`node -v`): v12.20.1
 - npm or yarn version (`npm -v`): v6.14.10
 - whatsapp-web.js version: v1.12.5

**Comment by stefanfuchs** [comment]
I tested in two different machines (one is an EC2 machine, the other is a docker image) and the same error occurs in both.

**Comment by gsrocha** [comment]
This is happening with me too, when i scan the qr code, the whatsapp say that this qr code don't come from web.whatsapp, and the code still come repeated

**Comment by stefanfuchs** [comment]
This message apeears to me too, in the mobile when scanning the qr code: "This qr code doesn't come from web.whatsapp.com"

**Comment by gsrocha** [comment]
I solve my trouble restarting my heroku node server. (Restart all dynos)
![image](https://user-images.githubusercontent.com/27821505/115452074-55e38b80-a1f4-11eb-8ca3-b99c77936220.png)

**Comment by frmontini** [comment]
Same here :(
Try to change Client.js on line 119:

`var QR_RETRY_SELECTOR = 'div[data-ref] > span > button';`

**Comment by stefanfuchs** [comment]
@gsrocha For me the problem persists after restarting the server.
@frmontini I'll try your solution. Did it work for you?

**Comment by frmontini** [comment]
@stefanfuchs Nops! I'm thinking to destroy the client when qr repeats and initialize again after that.
Until we find a solution :(

**Comment by stefanfuchs** [comment]
@frmontini actually, your solution worked for me. 
The problem was because the retry button changed in whatsapp web from `div` to a `button`
Now the qr codes seem to be being received correctly.

**Comment by pedroslopez** [comment]
Thanks for reporting the issue! I've issued a fix in f1e2f32988f501d8ecc9352690087531bbd793da and will release soon

**Comment by pedroslopez** [comment]
v1.12.6 has now been released
